src/acl.c

```diff
@@ -1254,9 +1254,85 @@ int ACLCheckCommandPerm(client *c, int *keyidxptr) {
     return ACL_OK;
 }
 
+int ACLCheckPubsubChannelPerm(sds channel, list *allowed, int literal) {
```

Comment:
please add a top comment that describes the inputs and output for faster read.

Reply:
Done.